### PR TITLE
PMM-7 migrate client build job to new farm

### PR DESCRIPTION
**Pull request by nikita-b**
Migrate job to new docker agents

### CODE REVIEW

File: pmm/pmm2-client-autobuild.groovy [hunk 1]
```diff
@@ -87,71 +79,64 @@ pipeline {
             steps {
                 withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
                     sh """
-                        sg docker -c "
-                            docker login -u "${USER}" -p "${PASS}"
-                        "
+                        docker login -u "${USER}" -p "${PASS}"
```
Comment (atymchuk) [inline pmm/pmm2-client-autobuild.groovy:82]: we'd better use `--password-stdin` as recommended by docker docs.

Comment (atymchuk) [inline pmm/pmm2-client-autobuild.groovy:82]:
```suggestion
                        echo "${PASS}" | docker login -u "${USER}" --password-stdin
```

File: pmm/pmm2-client-autobuild.groovy [hunk 2]
```diff
@@ -87,71 +79,64 @@ pipeline {
...
-                sh '''
-                    sg docker -c "
-                        env
-                        ./build/bin/build-client-rpm centos:7
-                        ./build/bin/build-client-rpm centos:8
-                    "
-                '''
+                sh '
+                    ./build/bin/build-client-rpm centos:7
+                    ./build/bin/build-client-rpm centos:8
```
Comment (atymchuk) [inline pmm/pmm2-client-autobuild.groovy:119]: Still not dropping centos:8?

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: pmm/pmm2-client-autobuild.groovy (hunk 2 above)
Comment (nikita-b) [reply pmm/pmm2-client-autobuild.groovy:119]: We can't drop it, because of RHEL 8 :(

**Review by atymchuk — APPROVED** [review]

**Pull request merged**